pir/cpp/client.h

```diff
@@ -58,7 +58,7 @@ class PIRClient {
   /**
    * Returns the database size.
    **/
-  std::size_t DBSize() const { return context_->DBSize(); }
+  std::size_t DBSize() const { return context_->Params()->database_size(); }
```

Comment:
Both client and server have this DBSize method. Perhaps this should be in Context? Why would a user of these classes need to access this method?